lib/targetctl.js

```diff
@@ -26,6 +26,19 @@ function getWorkerInfo() {
   };
 }
 
+function queryCapabilities(req, rsp, callback) {
+  var worker = getWorkerInfo();
+
+  rsp.cmd = 'query-capabilities';
```

Comment:
not necessary, the response is correlated with the request - it comes as a callback to the request() function,. `_appCommand` in your case.